lang/en.yml

```diff
@@ -379,7 +379,7 @@ en:
     PASSWORD: Password
     PASSWORDEXPIRED: 'Your password has expired. Please choose a new one.'
     PLURALNAME: Members
-    REMEMBERME: 'Remember me next time?'
+    REMEMBERME: 'Remember me next time? (for %d days on this device)'
```

Comment:
Agree with "Keep me signed in" label, it's more accurate. As the tooltip, I would suggest "Remember this device for %d days". Not too worried about exposing configuration. Showing the number of days helps users understand a) why they might be asked to log in again and b) increase confidence that logins won't work indefinitely. Then again, this might be such an established pattern that we don't need to be as explicit about it.

Facebook doesn't have a tooltip, but then again they have a lot more additional controls at work to ensure security. Basecamp says "Remember me on this computer" without a tooltip. Twitter just says "Remember me". Google has the following:
![image](https://cloud.githubusercontent.com/assets/111025/12876815/b0b02c8c-ce6c-11e5-8da8-b0d081d28071.png)

Here's a few more resources on the topic:
http://ux.stackexchange.com/questions/22216/do-we-really-need-a-remember-me-option-on-a-login-page
http://ux.stackexchange.com/questions/67539/remember-me-login-functionality?lq=1
http://www.siteflex.com.au/blog/n/remember-me-remember-me-not-140407
http://www.sitepoint.com/3-rules-painless-account-ux-login-screens/